command/helpers_test.go

```diff
@@ -685,3 +685,19 @@ func Test_extractVarFlags(t *testing.T) {
 		}, result)
 	})
 }
+
+func Test_extractJobSpecEnvVars(t *testing.T) {
+	ci.Parallel(t)
+
+	t.Run("nil", func(t *testing.T) {
+		must.MapEmpty(t, extractJobSpecEnvVars(nil))
+	})
+
+	t.Run("complete", func(t *testing.T) {
+		result := extractJobSpecEnvVars([]string{"NOMAD_VAR_count=13", "GOPATH=/Users/jrasell/go", "NOMAD_VAR_image=redis:7"})
```

Comment:
can you add a test where there is no value, e.g. `BLAH=` because that's totally a thing to be aware of in parsing env vars

Reply:
fun that you and I honed in on emptiness on opposite sides of the `=` :yum: